cluster/common.sh

```diff
@@ -286,6 +291,7 @@ function set_binary_version() {
   else
     KUBE_VERSION=${1}
   fi
+  export KUBE_VERSION
```

Comment:
do we actually want this exported? do other processes read it?
update the doc comment if so to clarify that it is exported

Reply:
Yes, this variable is used in hack/get-build.sh. I think it's better to export it.

Reply:
we should update the comment above to note that it is exported versus just set

Reply:
Yes and done.